src/sql/platform/accounts/common/accountViewModel.ts

```diff
@@ -48,23 +53,64 @@ export class AccountViewModel {
 		// 2) For each provider, get the accounts
 		// 3) Build parameters to add a provider and return it
 		try {
-			const metadata = await this._accountManagementService.getAccountProviderMetadata();
-			const accounts = await Promise.all(metadata.map(async providerMetadata => {
-				try {
-					const accounts = await this._accountManagementService.getAccountsForProvider(providerMetadata.id);
-					return <AccountProviderAddedEventParams>{
-						addedProvider: providerMetadata,
-						initialAccounts: accounts
-					};
-				} catch (err) {
-					this._logService.warn(`Error getting accounts for provider ${providerMetadata.id} : ${err}`);
-					return undefined;
-				}
-			}));
+			let accounts = await this.getAccountsForProviders();
+			if (this._authenticationService) {
+				const sessionAccounts = await this.getAccountForSessions();
+				accounts = accounts.concat(sessionAccounts);
+			}
+
 			return coalesce(accounts);
 		} catch (err) {
 			this._logService.warn(`Error getting account provider metadata : ${err}`);
 			return [];
 		}
 	}
+
+	private async getAccountsForProviders(): Promise<AccountProviderAddedEventParams[]> {
+		const metadata = await this._accountManagementService.getAccountProviderMetadata();
+		const accounts = await Promise.all(metadata.map(async (providerMetadata) => {
+			try {
+				const accounts = await this._accountManagementService.getAccountsForProvider(providerMetadata.id);
+
+				return <AccountProviderAddedEventParams>{
+					addedProvider: providerMetadata,
+					initialAccounts: accounts
+				};
+			} catch (err) {
+				this._logService.warn(`Error getting accounts for provider ${providerMetadata.id} : ${err}`);
+				return undefined;
+			}
+		}));
+		return accounts ?? [];
+	}
+
+	private async getAccountForSessions(): Promise<AccountProviderAddedEventParams[]> {
+		const sessionAccounts: AccountProviderAddedEventParams[] = [];
+		const providerIds = this._authenticationService.getProviderIds();
+
+		for (const providerId of providerIds) {
+			const displayName = this._authenticationService.getLabel(providerId);
+			const accountType = providerId === 'github' ? 'github' : 'microsoft'; // TODO: Find a better default account icon.
```

Comment:
'microsoft' account type logo is a generic outline of a person: 
![image](https://github.com/microsoft/azuredatastudio/assets/87730006/1b8b2067-2d1e-4e69-921e-e4542c610478)